### PR TITLE
Fix source date input selector and trimming month

### PR DESCRIPTION
**Pull request by jgoley**
Fixes [this bug](https://www.pivotaltracker.com/story/show/136507507) and I figured out a better way of changing the day abbreviations.

### CODE REVIEW

File: client/controllers/sourceModal.coffee [hunk 1]
```diff
@@ -192,7 +191,11 @@ Template.sourceModal.events
     input.value = @url
     titleInput = instance.find('#title')
     titleInput.value = @subject
+    $(input).trigger('input').trigger('input')
```
Comment (nathanathan) [inline client/controllers/sourceModal.coffee]: Why does this need to be triggered twice?

Comment (jgoley) [inline client/controllers/sourceModal.coffee]: I was wondering the same. I think it was originally introduced here: https://github.com/ecohealthalliance/eidr-connect/commit/8756b6f5a23b419ab4303c58bd4fb371ae414393#diff-96f74db91476ffbdb7217d25f79ce73bR180

Comment (jgoley) [inline client/controllers/sourceModal.coffee]: @nathanathan I went ahead and removed the second trigger method call.
This also fixes [the bug related to the source time not being autofilled](https://www.pivotaltracker.com/story/show/136737137)

**Comment by nathanathan** [comment]
👍

**Pull request merged**